### PR TITLE
WIP: Add threshold for errors before marking an incident as error or fixed

### LINKED ISSUE #74: Pagination for screenshots in UI Builds.
Display some items per page.

### PR DESCRIPTION
**Pull request by solker**
#74

### CODE REVIEW

File: shoov/modules/custom/shoov_ci_build/shoov_ci_build.features.field_base.inc [hunk 1]
```diff
@@ -236,5 +236,32 @@ function shoov_ci_build_field_default_field_bases() {
     'type' => 'text',
   );
 
+  // Exported field_base: 'field_status'
+  $field_bases['field_status'] = array(
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_ci_build/shoov_ci_build.features.field_base.inc]: => `field_ci_build_status`

**Comment by solker** [comment]
@amitaibu I've pushed. 

next step is check that messages not sent to users until finish state.

File: shoov/modules/custom/shoov_ci_build/shoov_ci_build.features.field_base.inc [hunk 2]
```diff
@@ -79,6 +79,33 @@ function shoov_ci_build_field_default_field_bases() {
...
+      'allowed_values' => array(
+        0 => 'new',
+        1 => 'done',
+        2 => 'unconfirmed error',
+        3 => 'error',
+      ),
+      'allowed_values_function' => '',
+    ),
+    'translatable' => 0,
+    'type' => 'list_integer',
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_ci_build/shoov_ci_build.features.field_base.inc]: lets use a non-required a list_text with the options:

```
unconfirmed error
error
```

We then can use NULL as indicate that it's ok.

Comment (amitaibu) [inline shoov/modules/custom/shoov_ci_build/shoov_ci_build.features.field_base.inc]: oh, I see you have `new` as-well

File: shoov/modules/custom/shoov_ci_build/shoov_ci_build.module [hunk 3]
```diff
@@ -6,6 +6,15 @@
...
 include_once 'shoov_ci_build.features.inc';
 
+// Possible statuses of CI Build.
+const CI_BUILD_STATUS_NEW = 0;
+const CI_BUILD_STATUS_DONE = 1;
+const CI_BUILD_STATUS_UNCONFIRMED_ERROR = 3;
+const CI_BUILD_STATUS_ERROR = 4;
+
+// The count of fails afters CI build set value to error.
+const CI_BUILD_MAXIMUM_UNCONFIRMED_ERRORS = 3;
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_ci_build/shoov_ci_build.module]: this should be a variable

File: shoov/modules/custom/shoov_ci_build/shoov_ci_build.features.field_base.inc (hunk 2 above)
Comment (solker) [inline shoov/modules/custom/shoov_ci_build/shoov_ci_build.features.field_base.inc]: @amitaibu yes, but really it's not a new. It's not prepared state.

**Pull request closed without merging**